apps/easypid/src/use-cases/ReceivePidUseCaseCFlow.ts

```diff
@@ -67,7 +76,7 @@ export class ReceivePidUseCaseCFlow extends ReceivePidUseCaseFlow {
         resolvedCredentialOffer: this.resolvedCredentialOffer,
         credentialConfigurationIdsToRequest,
         clientId: ReceivePidUseCaseCFlow.CLIENT_ID,
-        requestBatch: getShouldUseCloudHsm() ? 10 : false,
+        requestBatch: getShouldUseCloudHsm() ? 2 : false,
```

Comment:
Was this for testing?

Reply:
Need to update WSP. It doesn't batch soo is way too slow at 10